### DISCUSSION TITLE
Remove default image provider

### DISCUSSION THREAD

**Discussion by MarvinNazari** [gonzalezreal/swift-markdown-ui#298] [Ideas]
Amazing library! Thanks for putting the time to build this.
It seems like that this library has a direct dependency to [NetworkImage](https://github.com/gonzalezreal/NetworkImage). I would like to hold the numbers of my dependency down as much as possible and many projects already have a dependency for downloading and processing images or just it's good enough with a `AsyncImage` from `SwiftUI` it self.

So a solution would be to create a new target in the SPM Package that dosen't implement the default image provider which have NetworkImage as a dependency. If you are OK with that I could create a PR with it.

**Comment by gonzalezreal** [comment]
Hi @MarvinNazari,

Thanks for your kind words!

What's the issue with NetworkImage? It is a lightweight dependency maintained by me. To be honest, I don't see a compelling reason for creating a new target that doesn't use it. If having this dependency degrades build times or increases the binary size in unacceptable ways, I could change my mind.
